root/admin/components/UserList.js

```diff
@@ -65,6 +70,9 @@ const UserList = ({users}: Props): React.Element<'table'> => {
                 formatUserDate($c, user.last_login_date)
               ) : null}
             </td>
+            <td>
+              {nonEmpty(user.biography) ? user.biography : null}
```

Comment:
this will show raw HTML but I think that's fine on admin pages

Reply:
Yeah, that's what we show on the report :)